docs/resources/server/conditional.md

```diff
@@ -6,66 +6,93 @@ There are times when modders may want to include data-driven objects using infor
 Implementations
```

Comment:
If you're already changing this page, please migrate to # and ## for h1s and h2s.